lmdc/schemas/data_card_schema.yaml

```diff
@@ -0,0 +1,424 @@
+overview:
+  type: dict
+  schema:
+    article_link:
+      type: string
+      empty: True
+      nullable: True
+    name:
+      type: string
+      empty: False
```

Comment:
Does this mean that the field is required? Can we align terminology here? And is it just the dataset name that is the minimum requirement for the data card, or any other card?

Reply:
Yes.

So, each of the four cards has an Overview section which has the `name` field. This field should contain the name of the data/dataspace/model/service card. This is the only field that is required as of now.

Reply:
Aligning the terminology - if you mean rename `empty` to `required` that is not possible I guess. These are as per rules listed in cerberus.

https://docs.python-cerberus.org/validation-rules.html

Reply:
Actually, there's a rule for `required` as well.
I'll add it for the `name` fields in all cards.
`nullable` allows entering the field values as `None`. `empty` allows them to be left blank.

Reply:
Good, I only said that because "required" is the term we use on the paper and if people are not familiar with validation rules they might get confused.